src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h

```diff
@@ -317,13 +317,11 @@ namespace OpenMS
             throw Exception::IllegalArgument(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION,
               "Tried to calculate background estimation without any smoothed chromatograms");
           }
-          else if (background_subtraction_ == "original")
+          else
           {
-            calculateBgEstimationAverage_(used_chromatogram, best_left, best_right, background, avg_noise_level);
-          }
-          else if (background_subtraction_ == "exact")
-          {
-            calculateBgEstimationExact_(used_chromatogram, best_left, best_right, peak_apex_int, background, avg_noise_level);
```

Comment:
Double check that you updated for all places that this function is called (and the other functions) that are being replaced by PeakIntegrator.